client/src/components/SearchExperience.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import styled from "@emotion/styled";
+import StyledSpan from "./StyledSpan";
+import InputSlider from "./InputSlider";
+
+const Label = styled.label`
+  font-size: 1.3rem;
+  display: flex;
+  margin-top: 30px;
+  align-items: center;
+`;
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 60%;
+`;
+const Value = styled.span`
+  justify-content: flex-end;
+  display: flex;
+`;
+
+export default function SearchExperience({ value, onChange }) {
```

Comment:
`SearchExperience` wasnt clear for me until i read "Berufserfahrung"
Maybe `SearchJobExperience` or so?